src/Dijkstra/AST/Examples/Maybe/Branching.agda

```diff
@@ -4,14 +4,15 @@
    Licensed under the Universal Permissive License v 1.0 as shown at https://opensource.oracle.com/licenses/upl
 -}
 
+open import Dijkstra.AST.Prelude
 open import Data.Nat
-import      Level
-open import Util.Prelude hiding (bail)
 
 module Dijkstra.AST.Examples.Maybe.Branching where
 
 module Example-if (n : ℕ) where
   open import Dijkstra.AST.Maybe
+  open import Haskell.Prelude hiding (return)
```

Comment:
I think you mentioned these are to get functor, applicative, monad---right?

If that is correct, I suggest a qualified import `as HaskellPrelude` and using that prefix where needed.    Yes, ugly and long on purpose, since we really don't want to depend on `Haskell.Prelude`.

Reply:
Actually this import is no longer necessary.  FIXED.  Good catch, thanks.